gateway/handler.ts

```diff
@@ -1,29 +1,67 @@
 import { ApolloServer } from "apollo-server-lambda";
-import { createPastaporto } from "./pastaporto";
 import { gateway } from "./gateway";
-import httpHeadersPlugin from "apollo-server-plugin-http-headers";
-import cookie from "cookie";
+import { apolloHeadersPlugin } from "./plugins/apollo-headers";
+import { createContext } from "./context";
 
 const server = new ApolloServer({
   gateway,
   subscriptions: false,
   introspection: true,
-  plugins: [httpHeadersPlugin],
+  plugins: [apolloHeadersPlugin(true)],
   context: async ({ event }) => {
-    const cookieHeader = event.headers["cookie"];
-    let identity = null;
+    return createContext(event.headers["cookie"]);
+  },
+});
 
-    if (cookieHeader) {
-      const cookies = cookie.parse(event.headers["cookie"]);
-      identity = cookies["identity"];
+const handleManyCookies = (headers: any = {}) => {
```

Comment:
what's this for? can we add some comments?😊

Reply:
done